stan/math/opencl/matrix_cl.hpp

```diff
@@ -332,28 +332,47 @@ class matrix_cl<T, require_arithmetic_t<T>> {
 
   /**
    * Constructor for the matrix_cl that
-   * creates a copy of a scalar on the OpenCL device.
+   * creates a copy of a rvalue reference
+   * of a scalar on the OpenCL device.
```

Comment:
This change in the documentation is wrong. Please revert. `Scal&&` here is not necesarilly a rvalue reference. It is so-called forwarding reference, which can be deduced into either rvalue reference or lvalue reference.

Reply:
Yep, this is leftover from the way it was setup before with 2 separate constructors. Will fix in a minute. Tnx

Reply:
I just removed the "of a rvalue reference" part of the text. Is that ok with you?